File: data/imported/knowledge/slides/net/merged/snippets/snippet_022.cs
[Fact]
    public void TestCustomStringProperty()
    {
        // Custom string properties persist.
        using var pres = new Presentation();
        pres.DocumentProperties.SetCustomPropertyValue("MyProp", "hello");

        using var pres2 = TestHelpers.SaveAndReopen(pres, _tempDir);
        var value = pres2.DocumentProperties.GetCustomPropertyValue("MyProp");
        value.Should().Be("hello");
    }

File: data/imported/knowledge/slides/java/merged/snippets/snippet_022.java
@Test
    void testCustomStringProperty() throws IOException {
        try (var pres = new Presentation()) {
            pres.getDocumentProperties().setCustomPropertyValue("MyProp", "hello");

            try (var pres2 = saveAndReopen(pres)) {
                List<Object> out = new ArrayList<>();
                out.add(null);
                pres2.getDocumentProperties().getCustomPropertyValue("MyProp", out);
                assertThat(out.get(0)).isEqualTo("hello");
            }
        }
    }